### PR TITLE
✨ Authorization Service Layer Handlers feature

### PR DESCRIPTION
**Pull request by Khotso-Bore**
The service layer and it's accompanying Queries and handler functions have been completed. closed

### CODE REVIEW

**Comment by Ray-net** [comment]
Cant merge untill the linting and the build part are fixed

**Comment by Ray-net** [comment]
You need to resolve the error in the build test

**Comment by Ray-net** [comment]
Went from 5 errors to 2 almost there

**Comment by Ray-net** [comment]
Looks like a problem in the company explore

**Comment by Ray-net** [comment]
Might want to let then know in group chat

**Comment by Ray-net** [comment]
Cant commit till their work is fixed

**Comment by Khotso-Bore** [comment]
yeah, I was gonna suggest I wait till they put up a new PR before I try again

**Comment by Ray-net** [comment]
create test for api-authorization-api-feature:test
and api-authorization-service-feature:test

**Comment by Ray-net** [comment]
I will add the last test you figure out the linting problem

**Comment by Khotso-Bore** [comment]
i got the last test working
I just don't get the linting

**Comment by Ray-net** [comment]
Does one of you functions have a chance of returning undefined

**Comment by Ray-net** [comment]
If it does change it to return null

**Comment by Khotso-Bore** [comment]
none of them return anything undefined as far as i'm concerned

**Comment by Vhulenda88** [comment]
@arneschreuder, may you please assist us with the lint test failing, we are unaware of how to fix it

File: libs/api/authorization/api/feature/src/lib/api-authorization-api-feature.module.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import { Module } from '@nestjs/common';
 import { ApiAuthorizationServiceFeatureModule } from '@graduates/api/authorization/service/feature';
 import { ApiAuthorizationResolver } from './api-authorization.resolver';
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
```
Comment (arneschreuder) [inline libs/api/authorization/api/feature/src/lib/api-authorization-api-feature.module.ts:4]: Please remove and import correctly using @graduates/whatever notation. If this doesnt work. Your lib was not created correctly

File: libs/api/authorization/api/feature/src/lib/api-authorization-api-feature.module.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+import { ApiAuthorization } from '../../../shared/src/lib/api-authorization.entity';
```
Comment (arneschreuder) [inline libs/api/authorization/api/feature/src/lib/api-authorization-api-feature.module.ts:5]: Please import correctly. Use @graduates/whatever notation

File: libs/api/authorization/api/feature/src/lib/api-authorization-api-feature.module.ts [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
 @Module({
   controllers: [],
-  providers: [ApiAuthorizationResolver],
+  providers: [ApiAuthorizationResolver, ApiAuthorization],
```
Comment (arneschreuder) [inline libs/api/authorization/api/feature/src/lib/api-authorization-api-feature.module.ts:9]: I can’t remember that you have to provide entities. Please double check, but I think this is wrong

File: libs/api/authorization/api/feature/src/lib/api-authorization.resolver.spec.ts [hunk 4]
```diff
@@ -1,13 +1,21 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { ApiAuthorizationResolver } from './api-authorization.resolver';
-import { ApiAuthorizationService } from '@graduates/api/authorization/service/feature'
+import { ApiAuthorizationService } from '@graduates/api/authorization/service/feature';
```
Comment (arneschreuder) [inline libs/api/authorization/api/feature/src/lib/api-authorization.resolver.spec.ts:3]: This is wrong

**Review by arneschreuder — CHANGES_REQUESTED** [review]
Please fix imports, then resubmit for review. Also, please do not make the PR from your development branch, please make it from a feature branch. It makes PRs a bit difficult

**Pull request closed without merging**